build.gradle

```diff
@@ -187,13 +187,7 @@ dependencies {
     integrationTestRuntimeOnly "io.confluent:kafka-json-serializer:$confluentPlatformVersion"
 
     integrationTestImplementation "org.apache.kafka:connect-runtime:$kafkaVersion"
-    integrationTestImplementation "com.fasterxml.jackson.core:jackson-annotations:$jacksonVersion"
-    integrationTestImplementation "javax.servlet:javax.servlet-api:$servletVersion"
```

Comment:
These (and the dependencies removed below) are all transitively pulled in by the `connect-runtime` dependency. If we're not explicitly declaring them here intentionally, we can remove them.

(I discovered this while investigating an issue that turned out to be unrelated.)